css/global.css

```diff
@@ -0,0 +1,3 @@
+.gv.zq.zr.zs.zt.zu.zv.zw {
+  display: none !important;
```

Comment:
This CSS isn't really working - the classes might change for each article. I'll have to investigate other ways to do this.